src/Features/Core/Portable/StackTraceExplorer/DotnetStackFrameParser.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.StackTraceExplorer
+{
+    internal sealed class DotnetStackFrameParser : IStackFrameParser
+    {
+        private const string StackTraceAtStart = "at ";
+        private const string StackTraceSymbolAndFileSplit = " in ";
+
+        /// <summary>
+        /// Tries to parse a StackFrame following convention from Environment.StackTrace
+        /// https://docs.microsoft.com/en-us/dotnet/api/system.environment.stacktrace has
+        /// details on output format and expected strings
+        /// </summary>
+        public bool TryParseLine(string line, [NotNullWhen(true)] out ParsedFrame? parsedFrame)
+        {
+            parsedFrame = null;
+
+            var atIndex = line.IndexOf(StackTraceAtStart);
```

Comment:
this would match something like `(Phat phat, ...)`

Reply:
Changed to a startswith of the string